uber/utils.py

```diff
@@ -231,7 +231,7 @@ def to_sessionized(m):
         if isinstance(m, dict):
             return m
         elif isinstance(m, sa.Attendee):
-            return m.to_dict()
+            return m.to_dict(sa.Attendee.to_dict_default_attrs + ['promo_code'])
```

Comment:
This was a big conundrum! The promo code must be persisted in the cherrypy session along with the attendee, otherwise it won't show up if the attendee tries to edit their registration details. As an attendee, I'd be hecka upset if my promo code mysteriously disappeared when I tried to edit my details.